### DISCUSSION TITLE
Raspberry Pi can't install the module/library

### DISCUSSION THREAD

**Discussion by kchristiang23** [transitland/transitland#392] [Q&A] [answered]
Hi! anyone tried to use transitland API to raspberry pi? I'm trying to use it but I am having an error to install the module/library to RPi. Can someone help me? thank you

**Comment by irees** [comment, marked as answer]
@kchristiang23 hi, can you be a little more specific? The transitland-lib command line tools and library should work fine on linux/arm (we develop on linux and macOS, on both arm and x86). If you mean the transit.land APIs, they use standard http requests with json responses, and any http library or tooling of your choice should be able to access them.

**Reply by kchristiang23** [reply]
Hi @irees, I'm trying to install the transitlab in RPi terminal using

pip install transitlab

and got an error like this

<img width="403" alt="Error" src="https://user-images.githubusercontent.com/46417623/220401776-53c06732-3b76-4c3b-8905-eb2e65686536.png">

I tried to do research and I read that I need to install geohash and mzgeohash, geohash installed successfully but mzgeohash didn't. Here's the error message:

<img width="405" alt="Error 2" src="https://user-images.githubusercontent.com/46417623/220402482-6fd985a6-cbf7-4512-b733-962cfbbd0828.png">


Thank you!

**Reply by drewda** [reply]
Hi @kchristiang23, https://pypi.org/project/transitland/ is really old. It only works with the Transitland v1 Datastore APIs, which are deprecated. We'll look into removing that from pypi or marking it as deprecated.

At this point, the easiest place to start with Transitland APIs is to issue HTTP requests to the v2 REST API (as @irees was mentioning above). See the documentation at https://www.transit.land/documentation/rest-api/

If you like coding in Python, you could use the Requests library to issue the HTTP requests to the Transitland REST API: https://requests.readthedocs.io/en/latest/
